e107_handlers/form_handler.php

```diff
@@ -7002,7 +7002,7 @@ public function renderListForm($form_options, $tree_models, $nocontainer = false
 				$parms = 'total='.$total;
 				$parms .= '&amount='.$amount;
 				$parms .= '&current='.$from;
-				if(deftrue('e_ADMIN_AREA'))
+				if(deftrue('e_ADMIN_AREA') || deftrue('e_ADMIN_UI'))
```

Comment:
@Jimmi08 Please test frontend and backend with only this: ```if(deftrue('e_ADMIN_UI'))``` and resubmit if there is no issue.